living_documentation_regime/README.md

```diff
@@ -159,33 +174,19 @@ The Living Documentation Generator in Living Documentation regime is designed to
 ### Index Page Example
 
 ```markdown
----
-title: Features
-toolbar_title: Features
-description_title: Living Documentation
-description: >
-  This is a comprehensive list and brief overview of all mined features.
-date: 2024-12-04
-weight: 0
----
-
-<h1>Feature Summary page</h1>
-
-Our project is designed with a myriad of features to ensure seamless user experience, top-tier functionality, and efficient operations.
-Here, you'll find a summarized list of all these features, their brief descriptions, and links to their detailed documentation.
+# Issue Summary page
```

Comment:
Why the meta data were removed?
Are you presenting the Preview of expected output or raw data?